frame/system/call_remark_weight.go

```diff
@@ -1,23 +1,17 @@
 // THIS FILE WAS GENERATED USING GOSEMBLE BENCHMARKING PACKAGE
-// DATE: 2024-02-08 12:21:26.168771 +0200 EET m=+34.595230251, STEPS: 50, REPEAT: 20, DBCACHE: 1024, HEAPPAGES: 4096, HOSTNAME: MacBook-Pro.local, CPU: Apple M2 Pro(10 cores, 3504 mhz), GC: extalloc, TINYGO VERSION: 0.31.0-dev, TARGET: polkawasm
+// DATE: 2024-02-27 09:04:21.843885 +0200 EET m=+92.346653293, STEPS: `5`, REPEAT: `2`, DBCACHE: `1024`, HEAPPAGES: 4096, HOSTNAME: MacBook-Pro.local, CPU: Apple M2 Pro(10 cores, 3504 mhz), GC: , TINYGO VERSION: , TARGET:
```

Comment:
Can we update all the weights with the new template (specifically for the steps, repeat, dbcache)
Shouldn't heappages be in `` as well?

Reply:
I see that the weight generation leaves empty lines, can we trim those?

Reply:
ty for noticing, i thought empty lines were handled by checking array length>0 within the template. Will fix that